opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
>
> By including a codec facility in the API, those ad hoc HTTP calls
> propagate the TraceContext (or SpanContext, here) from process to process
> without relying on knowledge of the particular OpenTracing implementation.
>
>
We are defining context as a split across exactly two buffers. If there is
indeed a lot of practice doing this, perhaps we can find maybe even 3
tracers doing this?

I'm sure I can talk at high level to someone and they'd agree that
propagating context implies serialization. Devil is in the details... Did
the parties you've spoken to nod that they want a list of 2 byte arrays?
This is the primary source of my concern with this issue. I'd rather leave
it unspecified that awkwardly not like most apis.